src/main/java/nl/fuchsia/exceptionhandlers/ExceptionHandlers.java

```diff
@@ -0,0 +1,19 @@
+package nl.fuchsia.exceptionhandlers;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.MethodArgumentNotValidException;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+import org.springframework.web.bind.annotation.ResponseStatus;
+import org.springframework.web.bind.annotation.RestControllerAdvice;
+
+@RestControllerAdvice
+public class ExceptionHandlers {
+
+    @ResponseStatus(HttpStatus.BAD_REQUEST)
+    @ExceptionHandler(MethodArgumentNotValidException.class)
+    public ErrorResponse handleMethodArgumentNotValidException(
+            MethodArgumentNotValidException validException
```

Comment:
exception is naar mijn idee voldoende i.p.v. validException.

Reply:
Aangepast, kun je weer naar kijken.

Reply:
Helemaal goed.